File: BAEKJOON/C/2490.c
#include <stdio.h>

int main () {
    char str[3] = "";
    int i, j, input, sum, print;
    char result[6] = {'D', 'C', 'B', 'A', 'E', '\n'};
    
    for(i = 0; i < 3; i++){
        sum = 0;
        for(j = 0; j < 4; j++){
            scanf("%d", &input);
            sum += input;
        }
        str[i] = result[sum];
    }
    
    for(print = 0; print < 3; print++){
        printf("%c\n", str[print]);
    }
}

File: 2학년1학기/BAEKJOON/python/2490.py
import sys

result = ['D', 'C', 'B', 'A', 'E']
answer = []

for i in range(3):
    stick = list(map(int, sys.stdin.readline().split()))
    answer.append(result[sum(stick)])

for i in answer:
    print(i)

#시간복잡도 O(1)